### ISSUE TITLE
Random drop reference images?

### ISSUE THREAD

**Issue by sunjian2015** [Phantom-video/Phantom#39]
Thank you for your excellent work.
In the inference code, i see the code
``` 
neg = self.model([torch.cat([latent[:,:-ref_latent_neg.shape[1]], ref_latent_neg], dim=1) for latent, ref_latent_neg in zip(latents, ref_latents_neg)], t=timestep, **arg_null)[0]
noise_pred = neg + guide_scale_img * (pos_i - neg) + guide_scale_text * (pos_it - pos_i)
```
I want to know whether the reference image needs to be discarded with a certain probability during training, just like processing text?

**Comment by TianxiangMa** [comment]
Yes, during the model training, we perform dropout on the reference images with a 10% probability.

**Comment by sunjian2015** [comment]
> Yes, during the model training, we perform dropout on the reference images with a 10% probability.

I understand, thank you for your reply

**Comment by sunjian2015** [comment]
> Yes, during the model training, we perform dropout on the reference images with a 10% probability.

Is it setting all reference pictures to all 0?
